R/lowIntFilter.R

```diff
@@ -55,13 +55,13 @@ lowIntFilter <- function(dgeObj,
     }
 
     counts <- getItem(dgeObj, "counts")
-    geneLength <- dgeObj$geneData$ExonLength
```

Comment:
This line of code overwrites the geneLength paramter. It looks like it's left here by mistake